resources/assets/coffee/react/_components/user-avatar.coffee

```diff
@@ -28,10 +28,19 @@ bn = 'avatar'
 
   className = "#{bn} #{modifiers}"
 
-  if props.user.id?
+  avatar =
```

Comment:
~~this should be inside `if props.user.id?`~~

Looking again, this whole thing can (probably) be rewritten into

```coffeescript
## might as well replace with PureComponent while at it.
modifiers = ...

element = div
props = className: ...

if @props.user.id?
  props.style = backgroundImage: "..."

  if ... currentUser ...
    element = a
    props.title = ...
    props.href = ...
else
  props.className += ...

element props
```